### DISCUSSION TITLE
Getting "zsh: exec format error:" after installation of ohmyzsh on mac

### DISCUSSION THREAD

**Discussion by laxmikantbpandhare** [ohmyzsh/ohmyzsh#9841] [Help]
**Describe the bug**
operator-sdk is stopped working after installation of ohmyzsh

**To Reproduce**
Steps to reproduce the behavior, for example:
1. Install Operator-sdk on mac (https://sdk.operatorframework.io/docs/installation/)
2. Try "operator-sdk version" command then it will work
3. Now, install ohmyzsh
4. Now try same command "operator-sdk version" then it will throw error "zsh: exec format error: operator-sdk" 

**Expected behavior**
operator-sdk command should work after installation of ohmyzsh command

**Screenshots and/or Recordings**
If applicable, add screenshots to help explain your  #problem.
You can also record an asciinema session: https://asciinema.org/

Working without ohmyzsh -

<img width="1446" alt="Screen Shot 2021-04-07 at 8 23 24 PM" src="https://user-images.githubusercontent.com/47066536/113964193-d8754f80-97df-11eb-8864-260d725a83eb.png">

Error after installtion of ohmyzsh -

<img width="563" alt="Screen Shot 2021-04-07 at 8 23 04 PM" src="https://user-images.githubusercontent.com/47066536/113964156-c5627f80-97df-11eb-83c6-4f93b0fc53ed.png">

**Self Check**

- Have you tried reaching out on the [Discord server](https://discord.gg/ohmyzsh)? 
  This can help cut down on filling up issues. We always have a few people
  online that are in a variety of timezones that are willing to help you!

- Also searching existing [GitHub Issues](https://github.com/ohmyzsh/ohmyzsh/issues?q=) might help you get quicker support

**Desktop (please complete the following information):**

 - OS / Distro: macOS
 - Latest ohmyzsh update?: [e.g. Yes]
 - ZSH Version: 5.7.1
 - Terminal emulator: xterm-256color

**Additional context**
Add any other context about the problem here. This can be themes, plugins, custom configs.

**Comment by mcornella** [comment]
Does this happen when starting zsh with `zsh -f`?

**Comment by mcornella** [comment]
Also what does `file $(command -v operator-sdk)` show?

**Comment by laxmikantbpandhare** [comment]
mymac-mac% file $**(command -v operator-sdk)**
**/Users/username/go/bin/operator-sdk: Mach-O 64-bit executable x86_64**

**Comment by laxmikantbpandhare** [comment]
https://github.com/ohmyzsh/ohmyzsh/issues/9796#issuecomment-815596762
I haven't tried this. I uninstalled ohmyzsh for now.
